AdapterInterface.php

```diff
@@ -53,28 +57,32 @@ public function createAutoRoute(UriContext $uriContext, $document, $tag);
      * Return the canonical name for the given class, this is
      * required as somethimes an ORM may return a proxy class.
      *
+     * @param string $className
+     *
      * @return string
      */
     public function getRealClassName($className);
 
     /**
-     * Return true if the content associated with the auto route
-     * and the given content object are the same.
+     * Compares the content associated with the auto route and the
+     * given content object.
+     *
+     * @param AutoRouteInterface $autoRoute
+     * @param object             $contentObject
      *
-     * @param RouteObjectInterface
-     * @param object
+     * @return bool True when the contents are equal, false otherwise
      */
     public function compareAutoRouteContent(AutoRouteInterface $autoRoute, $contentObject);
 
     /**
      * Attempt to find a route with the given URL
      *
-     * Note that the URI may not be the same as the URI in the URI context, 
+     * Note that the URI may not be the same as the URI in the URI context,
      * this will happen when the ConflictResolver is trying to find candidate
      * URLs for example.
      *
-     * @param string $uri The URI to find
-     * @param UriContext $uri The current URI context
+     * @param string     $uri        The URI to find
```

Comment:
I don't think we do it in general (I know I never do, mainly because it is only realies on IDE features/plugins). But if we either adopt StyleCI or a PHPCSFixer config (hopefully there will be a CLI for style ci soon) then this isn't an issue.